common/workunit/workunit.cpp

```diff
@@ -9397,6 +9827,112 @@ IWUGraphStats *CLocalWorkUnit::updateStats(const char *graphName, StatisticCreat
     return new CWuGraphStats(LINK(p), creatorType, creator, _wfid, graphName, subgraph);
 }
 
+bool CLocalWorkUnit::usingDedicatedLogFiles(const char *processType, const char *processName, const char *logSpec) const
+{
+    if (isEmptyString(processType) || (isEmptyString(processName) && isEmptyString(logSpec)))
+        return false;
+
+    IArrayOf<IConstWUProcessLogInfo> processLogs;
+    if (!isEmptyString(processName))
+        getWUProcessLogInfo(processType, processName, processLogs);
+    else
+    {
```

Comment:
When does this apply?
i.e. when is usingDedicatedLogFiles() called without a processName ?

should add a comment to explain.

Reply:
The existing ECLWatch retrieves thor master  log using log file name (with full path and no processName). I will add a comment.